css/css-backgrounds/box-shadow-calc.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<link rel="author" title="Antoine Quint" href="mailto:graouts@apple.com" />
+<link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#the-box-shadow" />
+<link rel="match" href="box-shadow-calc-ref.html" />
+<meta name="assert" content="Test checks whether box-shadow supports calc() values.">
+<style type="text/css">
+div {
+  width: 100px;
+  height: 100px;
+  background-color: blue;
+  box-shadow: calc(1em + 10px) calc(1em + 10px) calc(1em + 10px) black;
```

Comment:
You should probably explicitly specify a font and size so that 1em is the same across all browsers.

Reply:
It used to be documented that it was safe to assume a `16px` default font size in CSS/WPT reftests, though I can't find that documentation anymore...

Reply:
👍

Reply:
https://github.com/web-platform-tests/wpt/blob/7b0ebaccc62b566a1965396e5be7bb2bc06f841f/docs/writing-tests/assumptions.md seems to be it today